api/tests.py

```diff
@@ -904,3 +904,38 @@ def add(a,b):
             self.assertTrue(result.get('success'))
         else:
             self.assertEqual(response.data.get('status'), 'running')
+
+
+class CreateCourseTest(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.username = 'demo'
+        self.password = 'demo'
+        self.user = User.objects.create_user(username=self.username,
+                                             password=self.password)
+
+    def test_create_course(self):
+        # Given
+        quiz = {'description': 'Quiz1', 'creator': 'demo'}
+        LearningUnit = {'quiz': quiz, 'type': 'quiz', 'order': 1}
+        leaningModule = {
+            'name': 'LM1',
+            'description': 'module one',
+            'creator': 'demo',
+            'learning_unit': LearningUnit
+        }
+        data = {
+            'name': 'Python Test Api Course',
+            'creator': 'demo',
```

Comment:
why are you sending a `creator` in the post data for the quiz, learning module?
The creator is required only at the course level.

Reply:
LearningModule too have a creator field.

Reply:
I know but anyway creator will be added in the views for the learning module and quiz, so you don't need it in the post data.